Otec/NewsRepository.swift

```diff
@@ -60,7 +60,7 @@ class NewsRepository: CRUDRepository {
     }
     
     private func urlStringWithPath(path: String) -> String {
-        return "http://43e9c7c6.ngrok.io" + "/" + path
+        return Constants.CanillitaBackendURL + "/" + path
```

Comment:
Then I'd rather suggest to separate this method out into a different class. It seems it has nothing to do with the repository itself. It looks confusing belonging here. Would you come up with a solution? Ping me if you need help.